bindings/core/src/method/utils.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use derivative::Derivative;
+use iota_sdk::types::block::{
+    address::{dto::Ed25519AddressDto, AliasAddress},
+    output::{AliasId, NftId, OutputId},
+    payload::transaction::dto::{TransactionEssenceDto, TransactionPayloadDto},
+    signature::dto::Ed25519SignatureDto,
+    BlockDto,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::OmittedDebug;
+
+/// Each public utility method.
+#[derive(Clone, Derivative, Serialize, Deserialize)]
+#[derivative(Debug)]
+#[serde(tag = "name", content = "data", rename_all = "camelCase")]
+pub enum UtilityMethod {
```

Comment:
Either call it `UtilsMethod` or call the file `utility.rs`

Reply:
Renamed